### PR TITLE
User account front end presentation

### PR DESCRIPTION
**Pull request by rafapolmann**
Implementing UserAccount front-end:
- Listing for UserAccounts
- Add new user account (with Default Pass)
- Edit basic properties (just name and access level)
- Create basic edit endpoint controller.

Implementing new rules in UserAccountService:
- Verify if exists another account with same e-mail on add new UserAccount.
- Generate default password on add UserAccount.
- Validate if exists ticktes linked to account on remove UserAccount.

Creating a few tests to UserAccountService

### CODE REVIEW

File: Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/components/user/user-account-form/user-account-form.component.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.form-group-flex-row mat-checkbox {
+  margin-bottom: 15px;
```
Comment (Akauam) [inline Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/components/user/user-account-form/user-account-form.component.scss:2]: No need for this class, there is no checkbox on this form

File: Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/features/user/user-account-edit/user-account-edit.component.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    private route: ActivatedRoute,
+    private userAccountService: UserAccountService,
+    private toastr: ToastrService) { }
+
+  ngOnInit(): void {
+    const id = Number(this.route.snapshot.paramMap.get('id'));
+    this.loadSession(id);
+  }
+
+  async loadSession(userAccountId: number) {
```
Comment (Akauam) [inline Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/features/user/user-account-edit/user-account-edit.component.ts:25]: I think this method name should be loadUserAccount

**Review by Akauam — APPROVED** [review]

File: Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/components/user/user-account-form/user-account-form.component.scss (hunk 1 above)
Comment (rafapolmann) [reply Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/components/user/user-account-form/user-account-form.component.scss]: Resolved

File: Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/features/user/user-account-edit/user-account-edit.component.ts (hunk 2 above)
Comment (rafapolmann) [reply Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/features/user/user-account-edit/user-account-edit.component.ts]: Resolved

**Pull request merged**